lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -203,13 +203,14 @@ def main():
         with tables.open_file(args.output_file, mode='a') as output:
             params = output.root[dl1_params_lstcam_key].read()
 
-            for tab in [output.root[dl1_params_lstcam_key], output.root[dl1_images_lstcam_key]]:
+            tabs = [output.root[dl1_params_lstcam_key]]
```

Comment:
Why do we store this config in each of the tables? Why not just at the file root?

Reply:
Question for @Bultako I think... But we can just fix the problem here and leave that for another PR, right?

Reply:
I think this is unnecessary duplication of the same info and this introduced this issue in the first place. `output.root.attrs` will always exist.

Reply:
Letting the monitoring tables have their own provenance information at the node level does not mean not putting the information on how the hole file was create at the root.

Reply:
Sure.
In fact, the root level has already the global metadata info about software versions used in producing the **HDF5 file**, while the keys/tables have config and the software versions used in producing the **tables**